packages/core/src/components/tabs/_tabs.scss

```diff
@@ -94,8 +102,11 @@ $tab-indicator-width: 3px !default;
 
 .#{$ns}-tab {
   @include overflow-ellipsis();
+  align-items: center;
+  align-self: stretch;
   color: $pt-text-color;
   cursor: pointer;
+  display: flex;
```

Comment:
~It looks like this does cause some issues for existing code which is inserting tags into tab titles (red border added for extra clarity):~

Edit: nevermind, we're good. the `align-items` style seems to fix things

| Before | After (without `align-items: center`) | After (with `align-items: center`) |
| - | - | - |
| <img width="374" alt="image" src="https://user-images.githubusercontent.com/723999/221689463-4805582c-b481-4f09-a70b-5e204a280922.png"> | <img width="373" alt="image" src="https://user-images.githubusercontent.com/723999/221689522-0b1b6d05-ad1b-43d3-901d-fe6efe14ceab.png"> | <img width="376" alt="image" src="https://user-images.githubusercontent.com/723999/221692411-b94dffd6-6ef7-4f57-8bdb-9c0396dd4caa.png"> |


the code for this is something like:

```ts
<Tab
  title={
    <>
      Pull requests
      {" "}
      <span className={classNames(Classes.TAG, Classes.MINIMAL, Classes.ROUND)}>0</span>
    </>
  }
/>
```